portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -21,7 +21,8 @@
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // TODO: figure out if there is better way to delete entities
```

Comment:
Yeah looks like I did just have a misunderstanding of the PreparedQuery. There is a method on `DatastoreService` that will delete a bunch of keys so two loops are not needed. You could do something like this in a follow up PR:

```
Query query = new Query("Comment-" + page);
query.setKeysOnly();
DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
List<Key> keys = new ArrayList<>();
for (Entity entity : datastore.prepare(query).asIterable()) {
  keys.add(entity.getKey());
}
datastore.delete(keys);
```